docs: table suffix (#1614)
GreptimeTeam/docs: docs/user-guide/logs/pipeline-config.md

If no rules match, data is transformed by the current pipeline's
transformations.

## Table suffix

:::warning Experimental Feature
This experimental feature may contain unexpected behavior, have its functionality change in the future.
:::

There are cases where the user want to split and insert log data into different target table

> based on some certain values of input data. For example, the user want to divide and store the log data 

Reviewer: ```suggestion
based on some certain values of input data. For example, if you want to divide and store the log data 
```

If no rules match, data is transformed by the current pipeline's
transformations.

## Table suffix

:::warning Experimental Feature
This experimental feature may contain unexpected behavior, have its functionality change in the future.
:::

> There are cases where the user want to split and insert log data into different target table

Reviewer: ```suggestion
There are cases where you want to split and insert log data into different target table
```
